index.html

```diff
@@ -22,17 +22,21 @@ <h1 class="name">Anne Hölscher</h1>
     </p>
     <br />
     <div class="button-group">
-      <a href="https://de.wikipedia.org/wiki/Anne_Bonny" class="button"
+      <a href="#about-me" class="button"
         >Über mich</a
       >
-      <a href="https://giphy.com/gifs/chill-relax-chilling-jeCSe7Qqc7NwKUzxH7" class="button">Lebenslauf</a>
+      <a href="#lebenslauf" class="button">Lebenslauf</a>
       <a href="https://giphy.com/gifs/v6aOjy0Qo1fIA" class="button"
-        >Katzen-GIF</a
+        >Lieblings-GIF</a
       >
     </div>
     <br />
     <hr />
```

Comment:
Instead of working with `br` elements you can style your `hr` element and add a margin or work with flexbox for your `main` content.